staging/src/k8s.io/kubectl/pkg/util/completion/completion_test.go

```diff
@@ -354,6 +354,54 @@ func TestPodResourceNameAndContainerCompletionFuncJointFormTooManyArgs(t *testin
 	checkCompletion(t, comps, []string{}, directive, cobra.ShellCompDirectiveNoFileComp)
 }
 
+func TestPodResourceNameAndContainerPortCompletionFuncNoArgsPodName(t *testing.T) {
```

Comment:
Sorry to jump in but i think test tables would be a lot better than writing few test functions here.

Reply:
nit: yeah, agree that these could be test tables.

Reply:
Updated